Competition2017/src/competition/subsystems/shooter_belt/ShooterBeltsManagerSubsystem.java

```diff
@@ -18,18 +20,46 @@
         protected ShooterBeltSubsystem leftBelt;
         protected ShooterBeltSubsystem rightBelt;
         
+        protected PIDPropertyManager leftPIDValues;
+        protected PIDPropertyManager rightPIDValues;
+        
+        boolean invertLeft = false;
+        boolean invertLeftSensor = false;
+        
+        boolean invertRight = true;
+        boolean invertRightSensor = true;
+        
         protected int leftMotorIndex = 31;
         protected int rightMotorIndex = 24;
 
         @Inject
-        public ShooterBeltsManagerSubsystem(WPIFactory factory, XPropertyManager propManager, RobotAssertionManager assertionManager){
+        public ShooterBeltsManagerSubsystem(WPIFactory factory, XPropertyManager propManager, PIDFactory pidFactory){
             log.info("Creating ShooterBeltSubsystem");
-            createLeftAndRightBelts(factory, propManager, assertionManager);
+            
+            leftPIDValues = pidFactory.createPIDPropertyManager("LeftBelt", 0, 0, 0, 0);
```

Comment:
We need to make sure that these are changed before competition. Otherwise, losing properties means that our robot is useless.